copilot/web/manifest.yml

```diff
@@ -0,0 +1,50 @@
+# The manifest for the "web" service.
+# Read the full specification for the "Load Balanced Web Service" type at:
+#  https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/
+
+# Your service name will be used in naming your resources like log groups, ECS services, etc.
+name: web
+type: Load Balanced Web Service
+
+# Distribute traffic to your service.
+http:
+  # Requests to this path will be forwarded to your service.
+  # To match all requests you can use the "/" path.
+  path: '/'
+  # You can specify a custom health check path. The default is "/".
+  healthcheck: '/healthcheck'
```

Comment:
AWS `healthcheck` requests are now sent to this destination (the endpoint was already implemented in our [middleware](https://github.com/compilerla/pems/blob/main/pems/core/middleware.py#L10) 🙂)